### PR TITLE
Rename PowerShell bootstrap variable from proxyUri to octopusProxyUri

### PR DESCRIPTION
**Pull request by tw17**
Fixes https://github.com/OctopusDeploy/Issues/issues/6204

One of our customers has encountered an issue where their variable template named `ProxyURI` is interfering with our `proxyUri` variable in the bootstrap script. to avoid this I have simple renamed this variable to `ocotpusProxyUri`
Our usage of this variable is only local to the Initialize-ProxySettings cmdlet.

### CODE REVIEW

File: source/Calamari.Shared/Integration/Scripting/WindowsPowerShell/Bootstrap.ps1 [hunk 1]
```diff
@@ -569,13 +569,13 @@ function Initialize-ProxySettings() {
 		#system proxy		
 		if ($useDefaultProxy) {
 			# The system proxy should be provided through an environment variable, which has been used to initialize $proxyHost
-			if ($proxyUri -ne $null) {
+			if ($octopusProxyUri -ne $null) {
 				# isWindows was introduced in PSCore and is not available for standard PowerShell
-				if (-not (Test-Path variable:IsWindows) -or $isWindows){
+				if (-not ((Test-Path variable:IsWindows) -or ($isWindows))){
```
Comment (tw17) [inline source/Calamari.Shared/Integration/Scripting/WindowsPowerShell/Bootstrap.ps1]: This is an unrelated bug that I noticed while debugging this code.

**Comment by tw17** [comment]
[Calamari bootstrap.ps1 variables](https://trello.com/c/PXHiXimn/2948-calamari-bootstrapps1-variables)

File: source/Calamari.Shared/Integration/Scripting/WindowsPowerShell/Bootstrap.ps1 (hunk 1 above)
Comment (emilol) [reply source/Calamari.Shared/Integration/Scripting/WindowsPowerShell/Bootstrap.ps1]: Nice find! I'd be keen to see this tracked as a separate issue though, that outlines what the bug was to make it easier for future Tom and future Emily. I've only been here a couple of weeks but have already had to refer back to issues related to this check.
